### ISSUE TITLE
Command line interface

### ISSUE THREAD

**Issue by jakejscott** [anaisbetts/SassAndCoffee#6]
I would love to be able to use Sass and Coffee on a appharbor hosted site but they don't give you disk access and I am not sure of any other limitations there may be in generating files on appharbor.

What would be involved in building a command line tool that we can use to statically generate css or js files?

**Comment by jakejscott** [comment]
Maybe I will just use compass command line tool directly :)

https://github.com/chriseppstein/compass

**Comment by anaisbetts** [comment]
Are you sure that it fails? I've heard from other folks that it works...

## 

Paul Betts paul@paulbetts.org

**Comment by jakejscott** [comment]
Yeah I tried to do it today :) No luck

**Comment by anaisbetts** [comment]
Theres an option in AppHarbor UI to make App_Data writable - is that checked?

## 

Paul Betts

SENT FROM MY COMMODORE 64: RESPONSES MAY BE IN ALL CAPS

**Comment by jakejscott** [comment]
Yeah I just checked it.. I originally thought that this should just work as I knew appharbor had that writable App_Data directory.

**Comment by jakejscott** [comment]
I mean yes - I have always had that option checked...and it doesn't work.

**Comment by anaisbetts** [comment]
I believe I know your problem, can you check your coffee / sass files, hit F4 to bring up properties, then make sure the file is marked as Content? Otherwise AppHarbor won't deploy it.

## 

Paul Betts

SENT FROM MY COMMODORE 64: RESPONSES MAY BE IN ALL CAPS

**Comment by jakejscott** [comment]
Wahoo all working! Check it out here http://lootrecruit.apphb.com/ 

I just needed to make sure that I had referenced all the IronRuby Libs and all the Microsoft Scripting libs too.

Should we add a note on the homepage telling people that it works on AppHarbor and the couple gotchas that I ran into..
Make sure .coffee and .scss files are set to content as visual studio doesn't by default.
Make sure that the IronRuby and MS Scripting libs are also referenced. I think that mine were missing as I used the NuGet Uninstall-Package SassAndCoffee at some point and when I re-installed it the IronRuby libs were not referenced. I also had iron ruby installed on my machine so it was probably grabbing them from the gac or somewhere..
